README.md

```diff
@@ -2289,29 +2289,33 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='switch-never-default'></a>(<a href='#switch-never-default'>link</a>) **Never use the `default` case when `switch`ing over an enum.**
+* <a id='switch-never-default'></a>(<a href='#switch-never-default'>link</a>) When switching over an enum, generally prefer enumerating all cases rather than using the `default` case.
```

Comment:
Pragmatically, there are 6400+ `default:` cases in our codebase right now. I don't think we should remove all of these, or require that all of them manually suppress the linter. When auditing a reasonable number of these, I find a fair number that I think look totally reasonable as-is.

Reply:
I think that _eventually_ there will be a Swift linter that is able to be run on the entire codebase and thus is able to have more complete knowledge. Such a linter would be more able to detect that a value is an enumeration, and thus avoid over-enforcing. If you believe that too, then I feel like we should be skating to where the world is moving rather than relaxing rules. And if we wanted, we could play a part in making a more advanced linter like that a reality :)

Reply:
When chatting about this off of github.com, I also mentioned that I think it would be appropriate if you could configure the compiler to emit a warning when a `default:` case is used with a `switch` statement over an enumeration.